test/node/conversation-labels.js

```diff
@@ -155,4 +177,9 @@ describe('Labels', () => {
             }
         });
     });
-});
\ No newline at end of file
+});
+
+function sleep(time) {
```

Comment:
sleep is already defined at https://github.com/circuit/circuit-sdk/blob/master/test/helper.js#L11

use helper.sleep.

And your sleep really blocks the event loop. See the implementation in the helper.